maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/AbstractBndMavenPlugin.java

```diff
@@ -649,6 +664,25 @@ protected void reportErrorsAndWarnings(Builder builder) throws MojoFailureExcept
 		}
 	}
 
+	protected boolean isEmpty(File directory) {
+		if (directory == null || !directory.exists()) {
```

Comment:
Perhaps this should be `!directory.isDirectory()`?  Otherwise this method will return `false` if `directory` is an existing file.